CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## Unreleased
 
+### Fixed
+
+Dynamic view height (#) and scrolling issues
+
+## [0.18.2] - 2023-08-29
+
+### Changed
+
+- Error capturing for python project fires to editor-api rather than Sentry (#625)
+
+## Added
+
+- Mobile project details component
+
```

Comment:
```suggestion
### Added

- Mobile project details component
- Mobile navigation between code and output (#615)

### Changed

- Position of run button on mobile (#615)
- Save button styles (#633)
- Tweak tab colors (#634)

### Fixed

Dynamic view height and scrolling issues (#631)

## [0.18.2] - 2023-08-29

### Changed

- Error capturing for python project fires to editor-api rather than Sentry (#625)

```